src/dev/build/tasks/nodejs/node_download_info.ts

```diff
@@ -28,14 +28,16 @@
  * under the License.
  */
 
-import { basename } from 'path';
+import { basename, resolve } from 'path';
 import fetch from 'node-fetch';
 import semver from 'semver';
 
 import { Config, Platform } from '../../lib';
 
 const NODE_RANGE_CACHE: { [key: string]: string } = {};
 
+export const NODE14_FALLBACK_VERSION = '14.21.3';
```

Comment:
would it be over-engineering a `.node_version_14` ?

Reply:
I agree that we would rather have one source of truth. Let's do a quick-follow on this.